### PR TITLE
improve(V3): simulate relay with V3 fn

### PR DESCRIPTION
**Pull request by james-a-morris**
Spurred by https://github.com/across-protocol/frontend-v2/pull/980/files#r1518357166

### CODE REVIEW

**Review by dohaki — APPROVED** [review]
LGTM

File: api/_utils.ts [hunk 1]
```diff
@@ -520,18 +520,20 @@ export const getRelayerFeeDetails = async (
...
         destinationChainId,
         originChainId,
-        relayerFeePct: sdk.utils.bnOne,
-        realizedLpFeePct: sdk.utils.bnOne,
-        recipient: recipientAddress,
-        message: message ?? sdk.constants.EMPTY_MESSAGE,
-        quoteTimestamp: sdk.utils.getCurrentTime(),
-        originToken,
-        destinationToken,
+        quoteTimestamp: sdk.utils.getCurrentTime() - 60, // Set the quote timestamp to 60 seconds ago ~ 1 ETH block
```
Comment (nicholaspai) [inline api/_utils.ts:530]: Why -60 if previously it was just current time?

File: api/_utils.ts [hunk 2]
```diff
@@ -520,18 +520,20 @@ export const getRelayerFeeDetails = async (
...
+        inputToken: originToken,
+        outputToken: destinationToken,
+        fillDeadline: sdk.utils.bnUint32Max.toNumber(), // Defined as `INFINITE_FILL_DEADLINE` in SpokePool.sol
+        exclusiveRelayer: sdk.constants.ZERO_ADDRESS,
+        exclusivityDeadline: 0, // Defined as ZERO in SpokePool.sol
+        message: sdk.constants.EMPTY_MESSAGE,
```
Comment (nicholaspai) [inline api/_utils.ts:536]: why not do ` message ?? sdk.constants.EMPTY_MESSAGE` as it was before?

**Review by nicholaspai — COMMENTED** [review]
why are we changing the param values passed to the sdk?

File: api/_utils.ts (hunk 1 above)
Comment (james-a-morris) [reply api/_utils.ts:530]: Spurred here: https://github.com/across-protocol/frontend-v2/pull/980#discussion_r1518355843

**Review by nicholaspai — APPROVED** [review]

**Pull request merged**